docs/manuals/running_services/index.md

```diff
@@ -87,4 +104,7 @@ if __name__ == "__main__":
     tp.run(gui, rest, title="Taipy Demo")
 ```
 
-!!! note "As mentioned, Taipy Rest needs Taipy Core to function, when running the Taipy Rest service, Taipy Core will also run along with it. Therefore, Taipy Core will also be together with Taipy GUI and Taipy Rest in the example above."
+Taipy Rest needs Taipy Core to function, when running the Taipy Rest service, Taipy Core will
+also run along with it. Therefore, Taipy Core will also be together with Taipy GUI and Taipy Rest in
```

Comment:
I don't understand the fragment: "Therefore, Taipy Core will also be together with"